plugins/Stk/Mallets/Mallets.cpp

```diff
@@ -284,7 +287,7 @@ void MalletsInstrument::playNote( NotePlayHandle * _n,
 	}
 
 	int p = m_presetsModel.value();
-	
+
```

Comment:
```suggestion
```
I think this is redundant.

Reply:
Given the previous discussion, I think this can be resolved as well without action.